server/src/main/java/fr/uga/l3miage/tp1/exo2/models/Session.java

```diff
@@ -0,0 +1,22 @@
+package fr.uga.l3miage.tp1.exo2.models;
+
+import javax.persistence.*;
+import java.util.UUID;
+
+@Entity
+@Table(name="session_entity")
+public class Session {
+    @Id
+    UUID idSession;
+
+    String lastCommand;
+
+    String currentDir;
+
+    boolean lock;
+
+    @OneToOne()
+    @JoinColumn(name="user_id", referencedColumnName = "idUser")
+    User user;
+
```

Comment:
Dans ce cas la, le `@JoinColumn` n'est pas utile tu utilise l'attribut qui est préfixé de `@Id`, et donc c'est ce que fait le `mappedby`
```suggestion
    @OneToOne
    UserEntity user;
```